src/main/scripts/run_e2e_test_module.py

```diff
@@ -70,19 +73,18 @@ def run_shell_command(cmd):
     assert res.ok or print(res.text)
     res=requests.put(f"http://localhost:11015/v3/namespaces/default/artifacts/{plugin_name}/versions/{plugin_version}/properties", json=plugin_properties)
     assert res.ok or print(res.text)
+    
     # Run e2e tests
-    os.chdir("../../../e2e")
-    print("Preparing e2e framework")
-    run_shell_command("mvn clean install")
+    os.chdir("../..")
+    print("cwd:", os.getcwd())
+    print("ls:", os.listdir())
     print("Running e2e integration tests")
-    assertion_error = None
-    os.chdir(f"../plugin/{module_to_build}")
     try:
-        run_shell_command("mvn clean install -P e2e-tests")
+        run_shell_command(f"mvn install -pl {module_to_build} -amd -P e2e-tests")
```

Comment:
We already built the module in Line#51 ig, why do we need to install it again??

Reply:
mvn verify is used to run test now